backend/validation.py

```diff
@@ -1,25 +1,45 @@
+class LocationValidationException(Exception):
+    def __init__(self, message):
+        super().__init__(message)
+
+
+class UpcValidationException(Exception):
+    def __init__(self, message):
+        super().__init__(message)
+
+
+INCORRECT_FIELD_ERROR = 'Incorrect Amount of Fields'
```

Comment:
You could place these constants within their relevant classes

Reply:
Corrected